### PR TITLE
Fetch `results.zip` from r20201130 instead of the latest release

### PR DESCRIPTION
**Pull request by mguaypaq**
As a quick fix for the ReadTheDocs builds failing, since I don't know yet how to re-generate `results.zip` for the more recent releases.

Part of #268.

### CODE REVIEW

**Comment by mguaypaq** [comment]
The python lint check is failing, but it's because of many code style violations unrelated to this PR:
```
./docs/conf.py:21:5: E266 too many leading '#' for block comment
./docs/conf.py:26:5: E266 too many leading '#' for block comment
./docs/conf.py:37:5: E266 too many leading '#' for block comment
./docs/conf.py:42:5: E266 too many leading '#' for block comment
./docs/conf.py:50:128: E501 line too long (140 > 127 characters)
./docs/conf.py:55:128: E501 line too long (152 > 127 characters)
./docs/conf.py:57:5: E266 too many leading '#' for block comment
./docs/conf.py:65:5: E265 block comment should start with '# '
./docs/conf.py:86:1: E402 module level import not at top of file
./spinegeneric/cli/__init__.py:1:1: W391 blank line at end of file
./spinegeneric/cli/check_data_consistency.py:75:17: F841 local variable 'originalFilePath' is assigned to but never used
./spinegeneric/cli/check_data_consistency.py:77:52: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
./spinegeneric/cli/create_mosaic.py:9:128: E501 line too long (227 > 127 characters)
./spinegeneric/cli/create_mosaic.py:27:1: E402 module level import not at top of file
./spinegeneric/cli/create_mosaic.py:30:1: E402 module level import not at top of file
./spinegeneric/cli/create_mosaic.py:31:1: E402 module level import not at top of file
./spinegeneric/cli/create_mosaic.py:32:1: E402 module level import not at top of file
./spinegeneric/cli/create_mosaic.py:34:1: E402 module level import not at top of file
./spinegeneric/cli/create_mosaic.py:155:43: F821 undefined name 'slice_size'
./spinegeneric/cli/create_mosaic.py:157:13: F841 local variable 'slice_size' is assigned to but never used
./spinegeneric/cli/create_mosaic.py:180:53: E203 whitespace before ':'
./spinegeneric/cli/deface_spineGeneric_usingR.py:1:1: E266 too many leading '#' for block comment
./spinegeneric/cli/deface_spineGeneric_usingR.py:2:1: E266 too many leading '#' for block comment
./spinegeneric/cli/deface_spineGeneric_usingR.py:3:1: E266 too many leading '#' for block comment
./spinegeneric/cli/deface_spineGeneric_usingR.py:4:1: E266 too many leading '#' for block comment
./spinegeneric/cli/deface_spineGeneric_usingR.py:5:1: E266 too many leading '#' for block comment
./spinegeneric/cli/deface_spineGeneric_usingR.py:6:1: E266 too many leading '#' for block comment
./spinegeneric/cli/deface_spineGeneric_usingR.py:93:25: E722 do not use bare 'except'
./spinegeneric/cli/deface_spineGeneric_usingR.py:108:29: E722 do not use bare 'except'
./spinegeneric/cli/generate_figure.py:41:1: F401 'plotly.express as px' imported but unused
./spinegeneric/cli/generate_figure.py:308:16: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:353:25: W605 invalid escape sequence '\p'
./spinegeneric/cli/generate_figure.py:357:25: W605 invalid escape sequence '\p'
./spinegeneric/cli/generate_figure.py:409:12: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:411:12: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:413:12: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:415:12: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:417:12: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:419:12: E713 test for membership should be 'not in'
./spinegeneric/cli/generate_figure.py:464:1: C901 'output_text' is too complex (16)
./spinegeneric/cli/generate_figure.py:530:40: E712 comparison to True should be 'if cond is True:' or 'if cond:'
./spinegeneric/cli/generate_figure.py:601:1: C901 'generate_figure_metric' is too complex (14)
./spinegeneric/cli/generate_figure.py:805:9: F841 local variable 'index' is assigned to but never used
./spinegeneric/cli/generate_figure.py:839:9: F841 local variable 'ci' is assigned to but never used
./spinegeneric/cli/generate_figure.py:840:9: F841 local variable 'cov_intra' is assigned to but never used
./spinegeneric/cli/generate_figure.py:841:9: F841 local variable 'cov_inter' is assigned to but never used
./spinegeneric/cli/generate_figure.py:1161:9: F841 local variable 'reg_predictor' is assigned to but never used
./spinegeneric/cli/generate_figure.py:1231:1: C901 'main' is too complex (20)
./spinegeneric/cli/generate_figure.py:1320:62: E712 comparison to True should be 'if cond is True:' or 'if cond:'
./spinegeneric/cli/manual_correction.py:156:1: C901 'main' is too complex (19)
./spinegeneric/cli/package_for_correction.py:85:1: C901 'main' is too complex (12)
./spinegeneric/cli/params_checker.py:40:1: C901 'main' is too complex (17)
./spinegeneric/cli/populate_derivatives.py:81:9: F841 local variable 'file_out' is assigned to but never used
./spinegeneric/cli/qc_bids_deface.py:47:13: W291 trailing whitespace
./spinegeneric/utils.py:51:17: F523 '...'.format(...) has unused arguments at position(s): 0
./spinegeneric/utils.py:80:60: E741 ambiguous variable name 'l'
./spinegeneric/utils.py:191:9: E722 do not use bare 'except'
```

**Comment by mguaypaq** [comment]
As discussed in the SCT developer meeting of 2023-09-07 ([minutes](https://docs.google.com/document/d/1ItApJQfajO2lRzOU2yenWbeRg6alfsdut3J4AVVdo78/edit#heading=h.cko7kqkgid08)), it's fine to pin `results.zip` to the release r20201130.

File: docs/conf.py [hunk 1]
```diff
@@ -39,13 +39,19 @@ def generate_html_figures(app):
...
+    # TODO: This is currently hard-coded to r20201130, rather than the latest release,
+    # to address https://github.com/spine-generic/spine-generic/issues/268
     path_zip_results_multisubject = os.path.join(path_data_stats, "results.zip")
+    # os.system(
+    #     'LOCATION=$(curl -s https://api.github.com/repos/spine-generic/data-multi-subject/releases/latest '
+    #     '| grep "tag_name" '
+    #     '| awk \'{print "https://github.com/spine-generic/data-multi-subject/releases/download/" '
+    #     'substr($2, 2, length($2)-3) "/results.zip"}\') '
+    #     '; curl -L -o ' + path_zip_results_multisubject + ' $LOCATION'
+    # )
```
Comment (joshuacwnewton) [inline docs/conf.py:51]: I don't think that this magic invocation is necessary anymore?

I think we can instead use https://github.com/spine-generic/data-multi-subject/releases/latest/download/results.zip once we start uploading `results.zip` again. ([source](https://stackoverflow.com/a/54836319))

**Review by joshuacwnewton — APPROVED** [review]
Thanks for this quick fix! :hearts:

File: docs/conf.py (hunk 1 above)
Comment (mguaypaq) [reply docs/conf.py]: Thanks for the tip!

**Pull request merged**